src/intl/messages.js

```diff
@@ -446,6 +446,8 @@ export const messages: { [messageId: string]: MessageType } = {
   nicNoVnicAssigned: 'N/A',
   nicsTooltip: 'Connected VM network interfaces.',
   noActiveStorageDomainInDataCenter: 'There is no active data storage domain in data center "{dataCenterName}"',
+  noClustersAvailable: 'No Clusters available',
+  noCdsAvailable: 'No CDs available',
```

Comment:
Could you, please, swap these two messages? To have them sorted nicely alphabetically ;) Thanks!

Reply:
Thanks, Done.